pkg/api/routes.go

```diff
@@ -712,6 +712,8 @@ func (rh *RouteHandler) UpdateManifest(response http.ResponseWriter, request *ht
 		return
 	}
 
+	rh.c.SubmitBackgroundTask(imgStore, name)
```

Comment:
I prefer calling out specific tasks. The call here could be a wrapper around it, for example, SubmitRepoTasks() etc which could schedule all tasks. More obvious, less magical.